spec/forms/keywords_form_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe KeywordsForm, type: :form do
+  describe '#save' do
+    context 'given a valid file it' do
+      it 'accepts a valid CSV' do
+        form = save_file('valid.csv')
+        expect(form.errors).to be_empty
+      end
+    end
```

Comment:
Should we rename the test a bit to follow [the convention](https://nimblehq.co/compass/development/code-conventions/ruby/rspec/#test-all-possible-cases)

```suggestion
    context 'given a valid file' do
      it 'does NOT add error' do
        form = save_file('valid.csv')
        expect(form.errors).to be_empty
      end
    end
```

Reply:
Updated in https://github.com/aetheryna/google-scraper-ruby/pull/34/commits/25b9f583ac59d64e0e10127480a5f768837eeeaa 😎